bitnami/appsmith/templates/_helpers.tpl

```diff
@@ -10,13 +10,27 @@ Return the proper Appsmith image name
 {{ include "common.images.image" (dict "imageRoot" .Values.image "global" .Values.global) }}
 {{- end -}}
 
+{{/*
+Return the proper Appsmith image name
+*/}}
+{{- define "appsmith.redirect.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.backend.redirectAmbassador.image "global" .Values.global) }}
```

Comment:
I think this image should be added to the `appsmith.imagePullSecrets` helper: https://github.com/bitnami/charts/blob/80e8c2eb7bc77b5726633cf9a2356df280a1f29a/bitnami/appsmith/templates/_helpers.tpl#L48-53